src/org/jetbrains/java/decompiler/main/extern/IFernflowerPreferences.java

```diff
@@ -387,6 +387,11 @@ public interface IFernflowerPreferences {
   @Type(Type.BOOLEAN)
   String REMOVE_IMPORTS = "remove-imports";
 
+  @Name("Mark Corresponding Sythetics")
```

Comment:
Typo: "sythetics"

Reply:
aah sorry, I caught myself making that typo a couple times but it wasn't enough apparently